<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-A13CRCOE" failures="1" tests="1" name="TwoTest" time="6.866" errors="0" timestamp="2023-10-03T19:00:56 IST">
  <testcase classname="testCases.TwoTest" name="testtwo" time="6.429">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TwoTest.testtwo(TwoTest.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testtwo -->
</testsuite> <!-- TwoTest -->
